src/Components/CreateImageWizardV2/steps/Oscap/index.tsx

```diff
@@ -10,9 +10,14 @@ import { useAppSelector } from '../../../../store/hooks';
 import { selectDistribution } from '../../../../store/wizardSlice';
 
 const OscapStep = () => {
-  const prefetchOscapProfile = imageBuilderApi.usePrefetch('getOscapProfiles');
+  const prefetchOscapProfile = imageBuilderApi.usePrefetch(
+    'getOscapProfiles',
+    {}
+  );
   const release = useAppSelector(selectDistribution);
-  prefetchOscapProfile({ distribution: release });
```

Comment:
prefetching in the top level of the component is a bad practice, it might triggered every re-render.
I guess a better prefetch would need here, not for the profiles list, but for the profiles details.

Reply:
I support removing pre-fetching for now and adding back later in a very specific PR.